.changes/unreleased/Removed-20240129-150301.yaml

```diff
@@ -0,0 +1,3 @@
+kind: Removed
```

Comment:
Can you change this to `Security` or `Fixed` please?

Removing an internal dependency shouldn't be a major release.

(I get that `Removed` makes sense, but it's for removing a feature, not removing a dep)

`.changie.yaml` explains the mapping